doc/developer/design/20230714_show_privileges.md

```diff
@@ -0,0 +1,202 @@
+# SHOW PRIVILEGES
+
+Associated:
+
+- [GitHub issue](https://github.com/MaterializeInc/materialize/issues/20452)
+- [RBAC design doc](20230216_role_based_access_control.md)
+- [System privilege design doc](20230531_system_privileges.md)
+- [Default privilege design doc](20230602_default_privileges.md)
+
+## Context
+
+Role based access control comprises a set of features that allow customers to control the
+privileges of individual users. Privileges can be granted to a user which allows them to take
+certain actions in a Materialize deployment. All objects have an owner, and certain actions are
+limited to only object owners. Roles can be members of other roles and inherit all of their
+privileges. Privileges, owners, and role membership are all stored within the database catalog.
+
+Currently, it's difficult for a user to discover RBAC related information.
+
+## Goals
+
+- Make it easy for users to discover the set of privileges that they have.
+- Make it easy for users to discover the owner of objects.
+- Make it easy for users to discover the roles that they are a member of.
+
+## Overview
+
+We will add the following SQL commands:
+
+- `SHOW PRIVILEGES`: shows the privileges of the current user.
+- `SHOW DEFAULT PRIVILEGES`: shows default privileges.
+- `SHOW ROLE MEMBERSHIP`
```

Comment:
Shouldn't this be in plural as well? `SHOW ROLE MEMBERSHIPS`?

Reply:
Updated.